discovery/discovery.yaml

```diff
@@ -42,10 +42,6 @@ paths:
       responses:
         '200':
           $ref: '#/components/responses/servicesdeleteresponse'
-        '400':
-          description: Bad Request - constraint failure
```

Comment:
I missed this.  I think we still want to reject any `id`s with bad formatting or any other request schema validation error.  Same below too.